Arduino/McLighting/McLighting.ino

```diff
@@ -692,6 +771,93 @@ void setup() {
     getStatusJSON();
   });
 
+  server.on("/pixels", []() {
+
+    bool updateStrip = false;
+    if(server.hasArg("ct")){
+      uint16_t pixelCt = server.arg("ct").toInt();
+      if (pixelCt > 0) {
+        WS2812FXStripSettings.stripSize = pixelCt;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.printf("/pixels: Count# %d\n", pixelCt);
+      }
+    }
+    if(server.hasArg("rgbo")){
+      String RGBOrder = server.arg("rgbo");
+      DBG_OUTPUT_PORT.print("/pixels: RGB Order# ");
+      if (RGBOrder == "grb") {
+        WS2812FXStripSettings.RGBOrder = NEO_GRB;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "gbr") {
+        WS2812FXStripSettings.RGBOrder = NEO_GBR;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "rgb") {
+        WS2812FXStripSettings.RGBOrder = NEO_RGB;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "rbg") {
+        WS2812FXStripSettings.RGBOrder = NEO_RBG;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "brg") {
+        WS2812FXStripSettings.RGBOrder = NEO_BRG;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "bgr") {
+        WS2812FXStripSettings.RGBOrder = NEO_BGR;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else {
+        DBG_OUTPUT_PORT.println("invalid input!");
+      }
+    }
+    if(server.hasArg("pin")){
+      uint8_t pin = server.arg("pin").toInt();
+      DBG_OUTPUT_PORT.print("/pixels: GPIO used# ");
+      #if defined(USE_WS2812FX_DMA) or defined(USE_WS2812FX_UART1) or defined(USE_WS2812FX_UART2)
+        #ifdef USE_WS2812FX_DMA
+          DBG_OUTPUT_PORT.println("3");
+        #endif
+        #ifdef USE_WS2812FX_UART1
+          DBG_OUTPUT_PORT.println("2");
+        #endif
+        #ifdef USE_WS2812FX_UART2
+          DBG_OUTPUT_PORT.println("1");
+        #endif
+      #else
+      if (pin == 16 or pin == 5 or pin == 4 or pin == 0 or pin == 2 or pin == 14 or pin == 12 or pin == 13 or pin == 15 or pin == 3 or pin == 1) {
+        WS2812FXStripSettings.pin = pin;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(pin);
+      } else {
+        DBG_OUTPUT_PORT.println("invalid input!");
+      }
+      #endif
+    }
+
+    if(updateStrip)
```

Comment:
Do we really need to check this? I mean: If nothing changed, we still could call these few lines. Nothing would happen, as nothing has changed...

Reply:
One never know what input one can receive. Always picture the user as a 4 yr old using a remote! (Ha ha) Let's say someone accidently uses GPIO10, one can fry the Flash. I did it in espeasy setup, that why I know! The web interface allowed any pin to be configured without these checks and magic smoke!